Yoast/javascript
# Add hungarian function words

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds function words for Hungarian and edits a typo in the Farsi function words file.
* [Yoast SEO Free] Improves keyphrase recognition in Hungarian by filtering the function words such as `harmadik`, `tiétek`, `valaki`.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

Test the following in WP SEO Free:

* Set the site language to Hungarian
* Create a new post and add a text with at least 300 words
* Set your keyphrase to `tizenkilenc macska`
* Add the word `macska` to the text
* Make sure that the keyphrase density shows that the keyphrase is found once

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-468
